### PR TITLE
#161047936 Create login page UI

### PR DESCRIPTION
**Pull request by boltonotieno**
#### What does this PR do?
 Creates a login page for  the web application
#### Description of Task to be completed?
Creation of a UI template that would allow users to log in using a username and a password
#### How should this be manually tested?
Clone my repository using git clone command: `git clone https://github.com/BoltC0rt3z/Store-Manager.git`
Go to the location to which you cloned the repository.
Change directory to Store Manager directory using` cd Store-Manager/UI`
Launch the login.html file using a  browser.
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[#161047936](https://www.pivotaltracker.com/story/show/161047936)
#### Screenshots (if appropriate)
login-page-screenshot

#### Questions:
What do you think about the UI?

### CODE REVIEW

**Comment by Mukuria-Samuel** [comment]
Design is generally good,
Just check on your link to pt story, the ID might be missing a number....its not an active link.

**Comment by waracci** [comment]
Great design :+1: I like the subtle box-shadow.

File: UI/login.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  
+        <label for="password"><b>Password:</b></label>
+        <input type="password" placeholder="Enter Password..." name="username" required>
+
+        <button type="submit">Login</button>
+    </form>
+    </div>
+    </div>
+</body>
+</html>
```
Comment (kwameasiago) [inline UI/login.html]: Its good practice to add a line at the end of your file before you push to git

**Comment by boltonotieno** [comment]
> Design is generally good,
> Just check on your link to pt story, the ID might be missing a number....its not an active link.

Okay on it... Thank you

**Comment by kwameasiago** [comment]
Beautiful User Interface

File: UI/login.html (hunk 1 above)
Comment (boltonotieno) [reply UI/login.html]: I'll do that.. Thank you

Comment (sharkdevs) [reply UI/login.html]: > 
> 
> Its good practice to add a line at the end of your file before you push to git

what do you mean by "add a line?"

**Comment by sharkdevs** [comment]
@BoltC0rt3z Good work. I would suggest you change the font to the Login form title to something like Roboto or Museo. Otherwise, Its just fine

**Comment by sylviawanjiku** [comment]
It looks simple and good

**Comment by Teatoller** [comment]
Hi BoltC0rt3z,
What a tough name you have, your design is simple and appealing to me.
Regards
Steven Ennis

File: UI/login.html (hunk 1 above)
Comment (boltonotieno) [reply UI/login.html]: What @SelaDanti mean is to add a new line at the end of  html file

**Comment by Paulvitalis200** [comment]
This is good. Maybe center the input fields for an even better design.

**Pull request merged**